storage/cache/cache.go

```diff
@@ -0,0 +1,71 @@
+package storage
```

Comment:
Is this module really used?

Reply:
I decided not to use cache functionality. Removed (a1f91c03963991b609ec707272670b421413cdd0).